src/plopp/core/__init__.py

```diff
@@ -7,3 +7,5 @@
 from .helpers import node, widget_node
```

Comment:
Can you then remove the `flake8` noqas above? I think they were mostly related to importing something but not using it.

Reply:
Yes, good find!